### ISSUE TITLE
A lot of DC issues

### ISSUE THREAD

**Issue by Shirieyn** [Soreepeong/XivAlexander#360]
I don't know what's wrong but it always disconnects me whenever I open Alex. Not immediately but at some point it will.

[XivAlexander_20220305_095535_416.log](https://github.com/Soreepeong/XivAlexander/files/8189624/XivAlexander_20220305_095535_416.log)

**Comment by Soreepeong** [comment]
You're using the official launcher, right? Not sure what's causing this problem, but as a workaround, you can try using [XIVLauncher](https://github.com/goatcorp/FFXIVQuickLauncher), or use XivAlexander as "Load once" mode instead of installing for now.

**Comment by Shirieyn** [comment]
Nah, I'm using XIVLauncher with Alex too. It never happened to me before until recently and I don't know what caused it.

**Comment by Soreepeong** [comment]
If that's the case, try turning off "Enable encrypting arguments to the client."

![image](https://user-images.githubusercontent.com/3614868/156901094-61b22830-de0a-4576-b66f-dee93c8fb595.png)

**Comment by Shirieyn** [comment]
Still the same thing. Tried reinstalling too for numerous times but to no avail. I ran an integrity check and this came up:
![image](https://user-images.githubusercontent.com/100980331/157901280-66220aab-2b45-45ea-be45-0f8db9c63a96.png)

**Comment by Soreepeong** [comment]
You should right click on the login button and click on "Repair", and let it proceed. You'll have to install TexTools mods again if you were using any, after repair is complete.
